migrate/migrate.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/scylladb/gocqlx/qb"
 )
 
+// ErrNoMigrationsFound is returned if the directory provided doesn't contain any migrations.
+var ErrNoMigrationsFound = errors.New("no migrations were found")
+
+// ErrNoCQLStmt is returned when trying to execute a non cql statement i.e. "This is not cql"
+var ErrNoCQLStmt = errors.New("no cql statement found")
```

Comment:
This error is a very rare case when you read empty files only. I suggest reusing ErrNoMigrationsFound.

Reply:
It could also be triggered if a file does not contain ";". In any case it is a rare occurrence, so I agree to use the same: errNoMigrations